### ISSUE TITLE
feature_importance_ missing features

### ISSUE THREAD

**Issue by fastaiml** [catboost/catboost#492]
Problem: ctb.CatBoostClassifier feature_importance_ returns fewer values than features in X_train. Tested over wide range of various tuning parameters (max_depth, n_estimators, border_count, loss_function, learning_rate, random_seed). Some combinations of these parameters with a fixed random_seed will return number of importance values equal to number of features, but the same parans with unspecified random seed will not (maybe yes sporadically). Tested on various platforms.

catboost version: 0.10.2
Operating System: Win (cuda 8.0), Debian (cuda 10)
CPU: Win Intel i7
GPU: Win (nvidia GeForce GTX 960M), Devian (K80, p100,)

**Comment by annaveronika** [comment]
If feature is not in feature_importance_ that means that this feature was not used in the model.
But you are right, we should assign a 0 value to it, we'll fix it.

**Comment by JustM57** [comment]
This doesn't depend on random seed. Always fix it while training models. The importance of the last feature in the list is missing(reasons are unknown).

**Comment by annaveronika** [comment]
The reason is that your last feature is never used in the model. Looks like it doesn't depend on the random seed

**Comment by annaveronika** [comment]
Actually this was fixed in 0.10.3
